### DISCUSSION TITLE
"Following" feed is empty

### DISCUSSION THREAD

**Discussion by 0x1eef** [community/community#35629] [Other Feature Feedback, Questions, & Ideas]
Hello !

I checked my "Following" feed today, and for some reason it is empty. I don't follow many people or projects, but I would expect to see some content - that's usually the case.

![emptyfeed](https://user-images.githubusercontent.com/8934693/194791559-5615956f-15c9-4016-8bac-a8143c3af7d5.png)

Best regards,

**Comment by jfullstackdev** [comment]
maybe you can use the other tab and check `Follows`, try other filters too

![image](https://user-images.githubusercontent.com/47092464/194880305-8bfdaa9d-24ab-44ea-be37-6bd84b5bf31c.png)

maybe those you follow does not have activities, that's why the default tab is empty

**Comment by 0x1eef** [comment]
@jdevstatic 

Thanks !! 
Yeah, that definitely helps but I still see follows from people I don't follow, and with the "Repositories" filter I still see activity from repositories I don't watch. It does include people, and repositories I do watch as well, though - so it is better than it currently is. To be honest I have lost hope that it will be fixed. It is the same since I reported the issue.
